### ISSUE TITLE
Issue with file writing/energy calculations

### ISSUE THREAD

**Issue by MeeMaster** [OpenMD/OpenMD#17]
I tried to run a simulation on a relatively small system, and encountered an error:

`StatWriter detected a numerical error writing: Total Energy`

The dump file stopped being written at some point. I attach the relevant files
[warm.zip](https://github.com/OpenMD/OpenMD/files/3916900/warm.zip)

**Comment by 123daimanu** [comment]
When I ran your system with the following option on metadata
`outputForceVector = "true";`
and checked the .eor file. I saw there were 4 atoms that had significantly larger force on it, which might because those atoms were very close to one another. The index of those atoms were 384, 391, 381, 375. Then I added a minimizer block in metadata (see in warmRelax.md) 
```
minimizer {
           useMinimizer = True;
           method = CG;
           maxIterations = 5000;
 }
```
and re-run the simulation. On looking though warmRelax.eor file, the larger force issue was solved. I used warmRelax.eor file to create the relaxed structure as warmAfterRelax.md. The system runs without any error. I have attached all relevant files below.

[warm.zip](https://github.com/OpenMD/OpenMD/files/3922445/warm.zip)
